src/node.h

```diff
@@ -349,8 +349,12 @@ inline std::unique_ptr<InitializationResult> InitializeOncePerProcess(
 }
 
 enum OptionEnvvarSettings {
-  kAllowedInEnvironment,
-  kDisallowedInEnvironment
+  kAllowedInEnvvar = 0,
```

Comment:
Are these options applicable only for the `NODE_OPTIONS` env var? If so, I wonder if `kAllowedInNodeOptions` is a better name.

Reply:
Thanks, this is a good idea! Updated, PTAL again :)

Reply:
The new name does seem pretty confusing unless you are familiar with the `NODE_OPTIONS` env var :)

Reply:
I concur. Perhaps `kAllowedInEnvOptions`?

Reply:
Are you suggesting `kAllowedInNodeOptions` is confusing? What do you think if we inline document the term `NodeOptions` in `kAllowedInNodeOptions`? Would that address your concerns?